### DISCUSSION TITLE
Inertia SSR with Docker

### DISCUSSION THREAD

**Discussion by azamat-747** [inertiajs/inertia#1930] [Help] [answered]
Hey Everyone!

I need deploy my project with Docker in VPS hosting,

my docker-compose.yaml

> version: "3.8"
> 
> services:
>   nginx:
>     image: "nginx:stable-alpine"
>     ports:
>       - "8000:80"
>     volumes:
>       - ./nginx/nginx.conf:/etc/nginx/conf.d/default.conf:ro
>       - ./src:/var/www/laravel
>     depends_on:
>       - mysql
>       - php
>   php:
>     build:
>       context: dockerfiles
>       dockerfile: php.Dockerfile
>     volumes:
>       - ./src:/var/www/laravel
>   mysql:
>     image: mysql:8.0
>     ports:
>       - "3316:3306"
>     env_file:
>       - env/mysql.env
>     volumes:
>       - ./db_data:/var/lib/mysql
>   npm:
>     build:
>       context: dockerfiles
>       dockerfile: node.Dockerfile
>     volumes:
>       - ./src:/var/www/laravel
>     ports:
>       - "5173:5173"
>   composer:
>     build:
>       context: dockerfiles
>       dockerfile: composer.Dockerfile
>     volumes:
>       - ./src:/var/www/laravel
>   artisan:
>     build:
>       context: dockerfiles
>       dockerfile: php.Dockerfile
>     volumes:
>       - ./src:/var/www/laravel
>     entrypoint: ["php", "/var/www/laravel/artisan"]
>   ssr:
>     build:
>       context: dockerfiles
>       dockerfile: php.Dockerfile
>     volumes:
>       - ./src:/var/www/laravel
>     command: php artisan inertia:start-ssr
>     ports:
>       - "13714:13714"

When i up project and run ssr, i see

Starting SSR server on port 13714...
Inertia SSR server started.

But SSR not working

**Comment by dairidong** [comment, marked as answer]
Set config `inertia.ssr.url` to `http://ssr:13714`.

**Reply by azamat-747** [reply]
Now it works, thanks
